Sources/Model/HyperwalletUser.swift

```diff
@@ -0,0 +1,847 @@
+            return self
+        }
+
+        /// Sets the user's state, province or region.
+        ///
+        /// - Parameter stateProvince: The user's state, province or region.
+        /// - Returns: a self reference of `HyperwalletUser.Builder` instance.
+        public func stateProvince(_ stateProvince: String) -> Builder {
+            return setField(key: UserField.stateProvince, value: stateProvince)
+        }
+
+        /// Sets the user account status.
+        ///
+        /// - Parameter status: The user's state, province or region.
+        /// - Returns: a self reference of `HyperwalletUser.Builder` instance.
+        public func status(_ status: Status) -> Builder {
+            return setField(key: UserField.status, value: status.rawValue)
+        }
+
+        /// Sets the local time of a region or a country. e.g. GMT, PST, ...
+        ///
+        /// - Parameter timeZone: The local time of a region or a country.
+        /// - Returns: a self reference of `HyperwalletUser.Builder` instance.
+        public func timeZone(_ timeZone: String) -> Builder {
+            return setField(key: UserField.timeZone, value: timeZone)
+        }
+        /// Sets the unique, auto-generated user identifier. Max 64 characters, prefixed with "usr-".
+        ///
+        /// - Parameter token: The unique, auto-generated user identifier. Max 64 characters, prefixed with "usr-".
+        /// - Returns: a self reference of `HyperwalletUser.Builder` instance.
+        public func token(_ token: String) -> Builder {
+            return setField(key: UserField.token, value: token)
+        }
+
+        /// Sets the the user's verification status. A user may be required to verify their identity after a certain
+        /// threshold of payments is reached.
+        ///
+        /// - Parameter verificationStatus: The user's verification status. A user may be required to verify their
+        ///                                 identity after a certainthreshold of payments is reached.
+        /// - Returns: a self reference of `HyperwalletUser.Builder` instance.
+        public func verificationStatus(_ verificationStatus: VerificationStatus) -> Builder {
+            return setField(key: UserField.verificationStatus, value: verificationStatus.rawValue)
+        }
+    }
+
+    private struct CodingKeys: CodingKey {
+        var intValue: Int?
+        var stringValue: String
+
+        init?(intValue: Int) {
+            self.intValue = intValue
+            self.stringValue = "\(intValue)"
+        }
+
+        init?(stringValue: String) {
+            self.stringValue = stringValue
+        }
+
+        static func make(key: String) -> CodingKeys {
+            return CodingKeys(stringValue: key)!
+        }
+    }
+
+    /// Gets the field value
+    ///
+    /// - Parameter fieldName: The `UserField` type
+    /// - Returns: Returns the field value, or nil if none exists.
+    private func getField(_ fieldName: UserField) -> Any? {
```

Comment:
It should be public for if integrator just wants to use this func to getField. Also after making it public move it before Builder.